com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Deferred/RaytracingGBuffer.raytrace

```diff
@@ -48,8 +48,6 @@ RW_TEXTURE2D_X(float4, _GBufferTexture0RW);
 RW_TEXTURE2D_X(float4, _GBufferTexture1RW);
 RW_TEXTURE2D_X(float4, _GBufferTexture2RW);
 RW_TEXTURE2D_X(float4, _GBufferTexture3RW); // Bake lighting and/or emissive
```

Comment:
Does it mean that raytracing don't support lightlayer and shadowmask anymore? (or it havee neverr supported it?)

Reply:
So it supported in quality mode. but not in performance mode. and yes it was not working, apparently I never did the full thing to make it work.